ocaml/xapi/xapi_vm.ml

```diff
@@ -693,9 +688,10 @@ let copy ~__context ~vm ~new_name ~sr =
   (* See if the supplied SR is suitable: it must exist and be a non-ISO SR *)
   (* First the existence check. It's not an error to not exist at all. *)
   let sr = try ignore(Db.SR.get_uuid ~__context ~self:sr); Some sr with _ -> None in
-  maybe (fun sr -> debug "Copying disks to SR: %s" (Db.SR.get_uuid ~__context ~self:sr)) sr;
+  Option.fold ~none:() ~some:(fun sr ->
```

Comment:
Why not `Option.iter` here and below?

Reply:
missed that function when reading the module :)